### DISCUSSION TITLE
Dúvida Simples (Codificação de erro)

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#625] [Tira Dúvidas] [answered]
Balta quando você menciona (vários cursos seus eu vi isso) lá na mensagem de um catch que você cria um código para o erro para facilitar, uma dúvida: você cataloga isso ou não?
Valeu

**Comment by andrebaltieri** [comment, marked as answer]
Sim! 

Talvez a maneira mais fácil de catalogar estes itens seja no projeto do GitHub (Por exemplo) na sessão Wiki, onde você pode colocar o código do erro, a descrição dele e como resolver.

A ideia é simples... o usuário recebe o erro #FC0128 por exemplo... ele abre um chamado informando este erro.
Com uma simples busca, pessoal de suporte consegue encontrar este código na Wiki e lá eles tem todas as informações relacionadas.

Além disso, como o código do erro é uma string `hard coded`, você pode pesquisar pelo código do erro no seu código fonte e vai encontrar o trecho exato onde ele ocorre 🙃

**Reply by FBODeveloper** [reply]
Beleza Balta.... muito obrigado
